.github/actions/init-npm/action.yaml

```diff
@@ -5,17 +5,25 @@ inputs:
     description: 'Node.js version'
     required: true
     default: '22.x'
+  skip-build:
+    description: boolean value wether should skip build strategy
+    required: true
+    default: false
 runs:
   using: composite
   steps:
+
   - name: Set up Node.js
     uses: actions/setup-node@v4
     with:
       node-version: ${{ inputs.node-version }}
       cache: npm
+
   - name: Install TS Project dependencies
     shell: bash
     run: npm ci
+
   - name: build
+    if: ${{ inputs.skip-build == 'false' }}
```

Comment:
have you tried it with skip-build as true? does it take the correct value? not by string?